completions/kldload

```diff
@@ -7,12 +7,16 @@ _kldload()
     local cur prev words cword
     _init_completion || return
 
-    local moddir=/modules/
-    [[ -d $moddir ]] || moddir=/boot/kernel/
+    local moddirs modules i
+    IFS=';' read -ra moddirs <<< "$(sysctl -n kern.module_path)"
```

Comment:
We avoid herestrings because they use temporary files under the hood. Maybe something like this (untested) instead?

```suggestion
    local moddirs modules i IFS=";"
    moddirs=($(sysctl -n kern.module_path))
    _comp_unlocal IFS
```